feather.go

```diff
@@ -14,11 +14,24 @@ func (r OnlineQueryBulkResult) Release() {
 	}
 }
 
-func (p OnlineQueryParamsComplete) ToBytes() ([]byte, error) {
+type SerializationOptions struct {
+	ClientConfigBranchId string
+}
+
+func (p OnlineQueryParamsComplete) ToBytes(options ...*SerializationOptions) ([]byte, error) {
+	branchId := p.underlying.BranchId
+	if len(options) > 1 {
+		return nil, fmt.Errorf("expected 1 SerializationOptions, got %d", len(options))
+	} else if len(options) == 1 {
+		if branchId == nil || *branchId == "" && options[0].ClientConfigBranchId != "" {
+			branchId = &options[0].ClientConfigBranchId
+		}
+	}
```

Comment:
is this behavior the same with python client? ie you can only override the branch if the client has no branch set?

Reply:
Ah I don't think that's the logic I intended here. Here I'm checking that if there is no branch override from the request params, then we set it to the client branch.